handshake_messages.go

```diff
@@ -1276,10 +1276,12 @@ func (m *newSessionTicketMsgTLS13) unmarshal(data []byte) bool {
 }
 
 type certificateRequestMsgTLS13 struct {
+	original                         []byte
 	ocspStapling                     bool
 	scts                             bool
 	supportedSignatureAlgorithms     []SignatureScheme
 	supportedSignatureAlgorithmsCert []SignatureScheme
+	certRequestCompressionAlgs       []CertCompressionAlgo
```

Comment:
It looks like you also have your second approach of fixing this from https://github.com/refraction-networking/utls/issues/345#issuecomment-2927991151 in here as well with the `original` and `originalBytes()`, could we remove the first approach if it works without it? It's a good reference in case we need to actually implement this in the future but I think your new approach is better.